src/packagers/yarn.ts

```diff
@@ -39,6 +39,14 @@ const getNameAndVersion = (name: string): { name: string; version: string } => {
  *   ignoreScripts (false) - Do not execute scripts during install
  */
 export class Yarn implements Packager {
+  private packagerOptions: PackagerOptions;
+
+  constructor(packagerOptions?: PackagerOptions) {
+    this.packagerOptions = packagerOptions || {
```

Comment:
What I would improve here is having this default option in the root configuration defaults
Here: https://github.com/floydspace/serverless-esbuild/blob/ec4aab91af3b50f32adb791be9e5399ca0c57228/src/index.ts#L279

Reply:
@floydspace Ah yeah, I had hesitated about doing this as it is only compatible with Yarn right now, as passing it for NPM wouldn't do anything. Instead setting the default in the `yarn.ts` file felt more _correct_. Happy to take your steer on it though, I can add it in here if you'd prefer

Reply:
--noInstall sounds generic. So potentially can be used for all packages.

Reply:
Yeah you're right – I guess I was _just_ doing the Yarn implementation in this PR to keep it small, assuming it could be easily extended by others as-needed.

I will make an additional commit that includes it in this list of defaults just so it's clear. At least a default of `false` is the standard existing functionality for all packagers

Reply:
Done in https://github.com/floydspace/serverless-esbuild/pull/421/commits/4b527a9497c43e9cf88796479bd3a9cc0d8825ca